core/src/main/scala/org/apache/spark/api/python/StreamingPythonRunner.scala

```diff
@@ -48,9 +48,8 @@ private[spark] class StreamingPythonRunner(func: PythonFunction, connectUrl: Str
    * Initializes the Python worker for streaming functions. Sets up Spark Connect session
    * to be used with the functions.
    */
-  def init(sessionId: String): (DataOutputStream, DataInputStream) = {
-    logInfo(s"Initializing Python runner (session: $sessionId ,pythonExec: $pythonExec")
-
+  def init(sessionId: String, workerModule: String): (DataOutputStream, DataInputStream) = {
+    log.info(s"Initializing Python runner (session: $sessionId ,pythonExec: $pythonExec")
```

Comment:
`logInfo`?

Reply:
Ah yea this is probably because of a concurrent PR of mine changing the same thing.. Thanks!